python/bids_import.py

```diff
@@ -271,8 +273,8 @@ def create_loris_bids_directory(bids_reader, data_dir, verbose):
 
     # determine the root directory of the LORIS BIDS and create it if does
     # not exist
-    name = re.sub("[^0-9a-zA-Z]+", "_", bids_reader.dataset_name) # get name of the dataset
-    version = re.sub("[^0-9a-zA-Z\.]+", "_", bids_reader.bids_version) # get BIDSVersion of the dataset
+    name = re.sub(r"[^0-9a-zA-Z]+", "_", bids_reader.dataset_name)       # get name of the dataset
```

Comment:
@regisoc was this change and the one below it on purpose?

Reply:
Hmm, no. I added on another branch but I am pretty sure I created this one based on main.

Reply:
okai but they are not accidental, thats the important part. If you can rebase and remove this commit `rebase -i` it would be better I think but @cmadjar will do the final review so it's her call

Reply:
Ok, there are 2 things here: 1/ the "r" intends to put the following string as a regex to nullify the backslash (else, flake8 raises a `W605` error) and 2/ it should be at least 2 spaces after for an inline comment (else, flake8 raises a `W261` error). But, neither `W605` nor `W261` are in the flake8 workflow exclusion https://github.com/aces/Loris-MRI/blob/06aa4125ac19f9f4601a8b1e7c0225f3140b8cae/.github/workflows/flake8_python_linter.yml#L38 So, do we add them?

Reply:
👀 huh?

@cmadjar ?

Reply:
yeah, those 2 changes are fine and compliant with flake 8 (the phpcbf of python)